File: 250318/배수의 개수 2/number-of-multipliers-2.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int count = 0;
        for (int i = 0; i < 10; i++) {
            int n = sc.nextInt();
            if(n % 2 == 1) {
                count++;
            }
        }
        System.out.println(count);
        sc.close();
    }
}

File: 250318/배수의 개수 2/number-of-multipliers-2.cpp
#include <iostream>
using namespace std;

int main()
{
    int count = 0;
    for(int i = 0; i < 10; i++) {
        int n = 0;
        cin >> n;
        if (n % 2 == 1) {
            count++;
        }
    }
    cout << count;
    return 0;
}